.github/workflows/prerelease.yml

```diff
@@ -47,46 +55,40 @@ jobs:
           account_id: ${{ secrets.COREINT_E2E_ACCOUNT_ID }}
           api_key: ${{ secrets.COREINT_E2E_API_KEY }}
           license_key: ${{ secrets.COREINT_E2E_LICENSE_KEY }}
-      - name: Create Release for the exporter modified
-        id: create_release
-        if: ${{ steps.check.outputs.CREATE_RELEASE == 'true'}}
-        uses: actions/create-release@v1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        with:
-          tag_name: ${{ steps.check.outputs.PACKAGE_NAME }}-${{ steps.check.outputs.VERSION }}
-          release_name: ${{ steps.check.outputs.PACKAGE_NAME }} ${{ steps.check.outputs.VERSION }}
-          body: |
-            Changes in ${{ steps.check.outputs.PACKAGE_NAME }} version ${{ steps.check.outputs.VERSION }}
-              - Exporter repository URL: ${{ steps.check.outputs.EXPORTER_REPO_URL }}
-              - Commit or Tag of the exporter packaged: ${{ steps.check.outputs.EXPORTER_HEAD }}
-              - CHANGELOG: ${{ steps.check.outputs.EXPORTER_CHANGELOG }}
-          prerelease: true
     outputs:
       CREATE_RELEASE: ${{ steps.check.outputs.CREATE_RELEASE }}
       EXPORTER_PATH: ${{ steps.check.outputs.EXPORTER_PATH }}
       PACKAGE_LINUX: ${{ steps.check.outputs.PACKAGE_LINUX }}
       PACKAGE_WINDOWS: ${{ steps.check.outputs.PACKAGE_WINDOWS }}
-      UPLOAD_URL: ${{ steps.create_release.outputs.upload_url }}
       INTEGRATION_NAME: ${{ steps.check.outputs.NAME }}
       INTEGRATION_VERSION: ${{ steps.check.outputs.VERSION }}
       RELEASE_TAG: ${{ steps.check.outputs.PACKAGE_NAME }}-${{ steps.check.outputs.VERSION }}
+      EXPORTER_REPO_URL: ${{ steps.check.outputs.EXPORTER_REPO_URL }}
+      EXPORTER_HEAD: ${{ steps.check.outputs.EXPORTER_HEAD }}
+      EXPORTER_CHANGELOG: ${{ steps.check.outputs.EXPORTER_CHANGELOG }}
 
   build_push_linux_artifacts:
     needs: [create_prerelease]
-    name: Push Linux Artifacts
+    name: Build and package package for Linux
```

Comment:
Nit: we could also rename the job. WDYT?

Reply:
Solved in [#a5426dd](https://github.com/newrelic/newrelic-prometheus-exporters-packages/pull/147/commits/a5426ddf7aae5d46b6419d29785cdb4550bab47a)